### PR TITLE
ci: do not throw away exit codes

### PR DESCRIPTION
**Pull request by simPod**
I could have noticed this sooner but we were throwing away test failures on unpatched versions - exit code was being overridden.

I tried to fix it in Make first but then decided to go with bash because of its flexibility (e.g. can define variables and traps).

### CODE REVIEW

File: bin/cleanup [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
```
Comment (greg0ire) [inline bin/cleanup:2]: Why not set `-eu` in all files?

Comment (simPod) [reply bin/cleanup:2]: It is inherited and controlled from the caller (`test-` files).

Comment (simPod) [reply bin/cleanup:2]: It is inherited and controlled from the caller (`test-` files).

File: tests/expected_report.txt [hunk 2]
```diff
@@ -35,19 +35,20 @@ tests/input/semicolon_spacing.php                     3       0
 tests/input/single-line-array-spacing.php             5       0
 tests/input/spread-operator.php                       6       0
 tests/input/static-closures.php                       1       0
+tests/input/strings.php                               1       0
```
Comment (greg0ire) [inline tests/expected_report.txt:38]: I doubt this has to do with your changes, right? If not, it should be in a separate commit.

Comment (simPod) [reply tests/expected_report.txt:38]: This PR fixes running tests on unpatched versions and `strings.php` is run there too. This was missing in the tests now, see https://github.com/doctrine/coding-standard/runs/7079914164?check_suite_focus=true

Comment (greg0ire) [reply tests/expected_report.txt:38]: Oh ok sorry I misunderstood the purpose of this PR

**Review by greg0ire — APPROVED** [review]

**Review by SenseException — APPROVED** [review]

**Review by Ocramius — APPROVED** [review]

**Comment by greg0ire** [comment]
Thanks @simPod !
